std/parallelism.d

```diff
@@ -2633,7 +2633,7 @@ public:
 
             // Hack to take the address of a nested function w/o
             // making a closure.
-            static auto scopedAddress(D)(scope D del)
+            static auto scopedAddress(D)(D del)
```

Comment:
The final version will be `return scope`. But the compiler isn't ready for that yet, so I put in this workaround for now.